### DISCUSSION TITLE
知识库向量化失败

### DISCUSSION THREAD

**Discussion by cyicz123** [lobehub/lobehub#4989] [General Question]
某一个文件分块后显示向量化失败，如下是报错。
![image](https://github.com/user-attachments/assets/b6e584b5-0cdd-49fa-b799-4b6c0e1b237f)

**Comment by arvinxx** [comment]
文件块太大了，你需要分下块

**Reply by cyicz123** [reply]
目前是lobechat进行分块的。请问我能否通过配置文件调整分块大小？

**Reply by arvinxx** [reply]
正常来说用 lobechat 常用的格式应该是不会遇到这个问题的？请问是什么格式的文件？以及不知是否方便将文件发出来我看下？

**Reply by cyicz123** [reply]
[2-2 迁移系统.pptx](https://github.com/user-attachments/files/18134233/2-2.pptx)
是pptx文件，一个课程ppt。请查收。
